runtime/bajun/src/lib.rs

```diff
@@ -22,6 +22,9 @@
 #[cfg(feature = "std")]
 include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 
+// Review comment: A lot o this code is copied from runtime/ajuna.
+// Maybe this duplication could be avoided by using feature toggles?
+
```

Comment:
Yeah we'll factor duplication out into `runtime/common` or something like that. Thanks for spotting this.